### DISCUSSION TITLE
Sphinx autosummary functionality

### DISCUSSION THREAD

**Discussion by ikamensh** [mkdocs/mkdocs#2821] [Q&A]
Hello,

I'd like to have simple documentation that mirrors the source code structure and just shows available modules / classes with their docstrings as HTML. I was a bit frustrated with Sphinx due to a bug, and also found it otherwise not intuitive and found Mkdocs as a recommended alternative.

Is there a feature to automatically generate docs from a source tree? if not, was this idea discussed previously?

**Comment by oprypin** [comment]
Check out https://mkdocstrings.github.io/recipes/

There is no integrated solution planned
